ruby/spec/session_spec.rb

```diff
@@ -146,6 +160,26 @@ def expect_request(body: nil, endpoint: nil)
       end
     end
 
+    describe "#http_client=" do
+      it 'uses provided HTTP Client' do
+        http_client = instance_double(Selenium::WebDriver::Remote::Http::Default)
+        session = Session.new()
+        session.http_client = http_client
+
+        expect(session.to_selenium[:http_client]).to eq http_client
+      end
+    end
+
+    describe "#listner=" do
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=5152675)